core/src/main/java/org/sonarsource/sonarlint/core/tracking/LocallyTrackedIssueTrackable.java

```diff
@@ -0,0 +1,108 @@
+/*
+ * SonarLint Core - Implementation
+ * Copyright (C) 2016-2023 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package org.sonarsource.sonarlint.core.tracking;
+
+import org.jetbrains.annotations.Nullable;
+import org.sonarsource.sonarlint.core.clientapi.backend.tracking.LocallyTrackedIssueDto;
+import org.sonarsource.sonarlint.core.commons.HotspotReviewStatus;
+import org.sonarsource.sonarlint.core.commons.IssueSeverity;
+import org.sonarsource.sonarlint.core.commons.RuleType;
+import org.sonarsource.sonarlint.core.commons.TextRangeWithHash;
+import org.sonarsource.sonarlint.core.issuetracking.Trackable;
+
+public class LocallyTrackedIssueTrackable implements Trackable {
+  private final LocallyTrackedIssueDto locallyTrackedIssue;
+
+  public LocallyTrackedIssueTrackable(LocallyTrackedIssueDto locallyTrackedIssue) {
+    this.locallyTrackedIssue = locallyTrackedIssue;
+  }
+
+  public LocallyTrackedIssueDto getLocallyTrackedIssue() {
+    return locallyTrackedIssue;
+  }
+
+  @Override
+  public Object getClientObject() {
```

Comment:
Is it supposed to represent the VirtualFile in the case of IntelliJ for example?

Reply:
It is the case for other trackables yes, here I return null because we don't need that. The Trackable interface will be reworked in a future increment